eng/Versions.props

```
@@ -6,15 +6,14 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>2</PatchVersion>
-    <SdkBandVersion>6.0.100</SdkBandVersion>
-    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
-    <PreReleaseVersionIteration>
-    </PreReleaseVersionIteration>
+    <SdkBandVersion>6.0.200</SdkBandVersion>
+    <PreReleaseVersionLabel>mauipre</PreReleaseVersionLabel>
+    <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
```

revert the branch specific changes:
```suggestion
    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
    <PreReleaseVersionIteration>
    </PreReleaseVersionIteration>
```